docs/languages/cpp.md

```diff
@@ -21,95 +21,112 @@ C/C++ support for Visual Studio Code is provided by a [Microsoft C/C++ extension
+## Example: Install MinGW-x64 on Windows
 
-## Example: Install MinGW-x64
+To understand the process, let's install Mingw-w64 via [MSYS2](https://www.msys2.org/). Mingw-w64 is a popular, free toolset on Windows. It provides up-to-date native builds of GCC, Mingw-w64, and other helpful C++ tools and libraries.
 
-We will install Mingw-w64 via [MSYS2](https://www.msys2.org/), which provides up-to-date native builds of GCC, Mingw-w64, and other helpful C++ tools and libraries.  You can download the latest installer from the MSYS2 page or use this [link to the installer](https://github.com/msys2/msys2-installer/releases/download/2022-06-03/msys2-x86_64-20220603.exe).
+1. Download using [**this direct link to the MinGW installer**](https://github.com/msys2/msys2-installer/releases/download/2023-05-26/msys2-x86_64-20230526.exe).
 
-Follow the **Installation** instructions on the [MSYS2 website](https://www.msys2.org/) to install Mingw-w64. Take care to run each required Start menu and `pacman` command.
+1. Run the installer and follow the steps of the installation wizard. Note, MSYS2 requires 64 bit Windows 8.1 or newer.
 
-You will need to install the full Mingw-w64 toolchain (`pacman -S --needed base-devel mingw-w64-x86_64-toolchain`) to get the `gdb` debugger.
+1. In the wizard, choose your desired Installation Folder. Record this directory for later. In most cases, the recommended directory is acceptable. The same applies when you get to setting the start menu shortcuts step. When complete, ensure the **Run MSYS2 now** box is checked and select **Finish**. A MSYS2 terminal window  will then automatically open.
 
-### Add the MinGW compiler to your path
+1. In this terminal, install the MinGW-w64 toolchain by running the following command:
 
-Add the path to your Mingw-w64 `bin` folder to the Windows `PATH` environment variable by using the following steps:
+    ```MSYS2
+    pacman -S --needed base-devel mingw-w64-x86_64-toolchain
+    ```
 
-   1. In the Windows search bar, type 'settings' to open your Windows Settings.
-   1. Search for **Edit environment variables for your account**.
-   1. Choose the `Path` variable in your **User variables** and then select **Edit**.
-   1. Select **New** and add the Mingw-w64 destination folder path, with `\mingw64\bin` appended, to the system path. The exact path depends on which version of Mingw-w64 you have installed and where you installed it. If you used the settings above to install Mingw-w64, then add this to the path: `C:\msys64\mingw64\bin`.
-   1. Select **OK** to save the updated PATH. You will need to reopen any console windows for the new PATH location to be available.
+1. Accept the default number of packages in the `toolchain` group by pressing `kbstyle(Enter)`.
 
-### Check your MinGW installation
+    ![MYSS2 Installer](/docs/cpp/images/cpp/cpp-install-MSYS2-toolchain.png)
```

Comment:
Better to duplicate the image under the `/images/cpp` folder

Reply:
fixed